docs/getting-started/dockstore-workflows.rst

```diff
@@ -114,31 +114,25 @@ This is the newest way of getting content onto Dockstore and is by far the most
 GitHub Apps, Dockstore can react to changes on GitHub as they are made, keeping Dockstore synced
 with GitHub automatically.
 
-To install our GitHub App on either a repository or organization,
-navigate to the ``/my-workflows`` page and click add workflow. Follow the steps for GitHub Apps and
-you will be redirected to GitHub where you can select which repositories to install the Dockstore
-GitHub app on. The process is shown in the following images.
+Installing the GitHub App is simply -- navigate to ``/my-workflows``, click the ``+`` button on the left hand sidebar, select ``Register using GitHub Apps``, and then click
+``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can select which repositories can be accessed by the GitHub app.
```

Comment:
Nit-picky, and maybe this is fine the way it is, but you are granting the GitHub app permissions to access your repo(s). I feel "select which... can be accessed" is a little passive.